File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.py
def solution(before, after):
    a = []
    for i in before:
        if before.count(i)==after.count(i):
            a.append(1)
        else:
            a.append(0)
    if a.count(1)==len(before):
        return 1
    else:
        return 0

File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.java
import java.util.Arrays;

class Solution {
    public int solution(String before, String after) {
        if (before.length() != after.length()) {return 0;}
        char[] c1 = before.toCharArray();
        char[] c2 = after.toCharArray();
        Arrays.sort(c1);
        Arrays.sort(c2);
        int answer = 0;
        if (Arrays.equals(c1, c2)) {
            answer = 1;
        }
        return answer;
    }
}